soft_top_speed: 4.0 # m/s
acceleration_interpolation_min: 2.5 # m/s^2
acceleration_interpolation_max: 7.5  # m/s^2
speed_at_acceleration_max: 10.0 # m/s
yaw_on: true
use_depth_image: true
flight_altitude: 1.5 # m
use_3d_library: false
max_e_stop_pitch_degrees: 45.0
laser_z_below_project_up: -0.5 # m
A_dolphin: 0.4 # m
T_dolphin: 3.0 # s
